test/Window/Keyboard.test.cpp

```diff
@@ -0,0 +1,54 @@
+#include <SFML/Window/Keyboard.hpp>
+
+// Other 1st party headers
+#include <SFML/System/String.hpp>
+
+#include <catch2/catch_test_macros.hpp>
+
+#include <WindowUtil.hpp>
+
+TEST_CASE("[Window] sf::Keyboard", runDisplayTests())
+{
+    SECTION("isKeyPressed(Key)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::D));
+    }
+
+    SECTION("isKeyPressed(Scancode)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::D));
+    }
+
+    SECTION("localize(Scancode)")
+    {
+        CHECK(sf::Keyboard::localize(sf::Keyboard::Scan::Space) == sf::Keyboard::Space);
+    }
+
+    SECTION("delocalize(Key)")
+    {
+        CHECK(sf::Keyboard::delocalize(sf::Keyboard::Space) == sf::Keyboard::Scan::Space);
+    }
+
+    SECTION("getDescription(Scancode)")
+    {
+        CHECK(sf::Keyboard::getDescription(sf::Keyboard::Scan::Space) == "Space");
```

Comment:
Descriptions are translated in user language at least on Windows. This test would fail then.

Reply:
Can I at least keep the function key tests or are those prone to fail as well?

Reply:
I guess function keys are a good choice for this test. I am not aware of a setup that would make the test fail with function keys.